{{cookiecutter.project_name}}/docker/django/Dockerfile

```diff
@@ -42,7 +42,15 @@ RUN chmod +x "/docker-entrypoint.sh" \
   && poetry config settings.virtualenvs.create false \
   && poetry install $(test "$DJANGO_ENV" == production && echo "--no-dev") --no-interaction --no-ansi
 
-# Creating folders, and files for a project:
-COPY . /code
+# This dir will become the mountpoint of development code
+WORKDIR /code
 
 ENTRYPOINT ["/sbin/tini", "--", "/docker-entrypoint.sh"]
+
+
+FROM development_build as production_build
```

Comment:
This requires two things:
1. Docks about multi-stage builds: a link to the official site
2. Link to our own docs: https://wemake-django-template.readthedocs.io/en/latest/pages/template/production.html

Reply:
ok done